lib/Runtime/Executor/ThreadPoolExecutor.cpp

```diff
@@ -101,8 +103,10 @@ ExecutionState::ExecutionState(RunIdentifierTy id, const DAGNode *root,
       const auto &symbolInfo = symbolPair.second;
 
       if (symbolInfo.symbolCategory == SymbolCategory::Placeholder) {
-        nodeInputPhBindings->allocate(
-            createOrGetPlaceholder(symbolName, &symbolInfo.type));
+        auto PH = module_->getPlaceholderByName(symbolName);
+        // If a PH name is provided it had to come from the Module originally.
+        DCHECK(PH) << "Placholder: " << symbolName << " is not in the module";
```

Comment:
nit: Placholder typo

Reply:
I like DCHECK but this is the first time we're using it over GLOW_ASSERT, right?

Reply:
@jackm321 beat me to it, it's already being used in Tensor.cpp as well.